src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator/press--options--.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'press(key, [options])'
+excerpt: 'xk6-browser: locator.press method'
+---
+
+Press a single key on the keyboard or a combination of keys.
+
+| Parameter | Type   | Description                                                                      |
+|-----------|--------|----------------------------------------------------------------------------------|
+| key       | string | Name of the key to press or a character to generate, such as `ArrowLeft` or `a`. |
```

Comment:
Should we have a reference page with the full list of valid key names?

Or even better if we can link to an existing one in some JS spec somewhere? We don't want to link to our [layout file](https://github.com/grafana/xk6-browser/blob/1997670b94d3252768c2a5da80b645957634be3b/keyboardlayout/us.go), and I'm pretty sure these are standardized somewhere.

Reply:
Yep, that's a good idea. I looked at the [PW docs](https://playwright.dev/docs/api/class-keyboard#keyboard-down) and they link to https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values. WDYT of using the same link?

Reply:
Resolved in: 625289795dcb9a13542f8db491eabca349df3c8b